### ISSUE TITLE
Controller Response Test Failure After Power Loss

### ISSUE THREAD

**Issue by HuggableShark** [thorrak/brewpi-esp8266#38]
My Wesmos D1 seems to be having issues persisting between fresh firmware flashes after a series of power losses I experienced recently. The primary indicator I'm noticing is the Controller Response Test is failing on both serial and WiFi connections, even though all other tests are showing that things are fine. The major symptom I'm noticing here though is that the relay switch is constantly  keeping both cooling and heating elements going simultaneously, regardless of what my Fermentrack installation tells it to try to do. I have confirmed that the controller is properly connected to my wifi and it is even keeping a constant log of my temperature sensor readings.

As previously mentioned, I have tried re-flashing the controller multiple times and even uninstalled/reinstalled the controller to Fermentrack just for good measure. Stderr logs are stating that both my Onewire temp sensors are constantly disconnecting, then reconnecting practically every second.

My pin layout is as follow on the controller:
D0 = Heat
D5 = Cool
D6 = Data/Sensors
5v  = Power
G   = Ground

Basically what you'd expect. I just can't seem to figure out why the controller would be sending data to the relay to power both elements at once. If you would like any more information or logs just let me know. I'd greatly appreciate some help solving this.

**Comment by HuggableShark** [comment]
After breaking the controller down and setting it back up again I noticed the inverted pins option is selected by default. It was set to standard pins during the issue. After flipping both sensors back to inverted pin status they worked appropriately. Issue solved!
